src/lib.rs

```diff
@@ -169,6 +169,28 @@ fn render_message(mut w: impl Write, msg: &Message, dbc: &DBC) -> Result<()> {
         )?;
         writeln!(w)?;
 
+        for signal in msg.signals().iter() {
+            let typ = signal_to_rust_type(&signal);
+            if typ != "bool" {
+                writeln!(
+                    &mut w,
+                    "pub const {sig}_MIN: {typ} = {min}_{typ};",
```

Comment:
I don't think we need the type suffix because we have the type ascription :)

Reply:
Without suffix there are problems with `0_f32`. For example this is not compiling:
```rust
pub const VOLTAGE_MIN: f32 = 0;
```

Reply:
About `63.9990234375_f32` yes you are right, it is taken from dbc without any conversion.

Reply:
Ah, of course, because there's no decimal point. Let's keep it like this :+1: